tasks/elasticsearch-parameters.yml

```diff
@@ -12,10 +12,7 @@
 #Check if working with security we have an es_api_basic_auth_username and es_api_basic_auth_username - otherwise any http calls wont work
 - name: fail when api credentials are not declared when using security
   fail: msg="Enabling security requires an es_api_basic_auth_username and es_api_basic_auth_password to be provided to allow cluster operations"
-  when:
-    - es_enable_xpack and "security" in es_xpack_features
-    - es_api_basic_auth_username is not defined
-    - es_api_basic_auth_password is not defined
+  when: not oss_version and es_api_basic_auth_username is defined and es_api_basic_auth_password is not defined
```

Comment:
Should
```
es_api_basic_auth_username is defined
```
be
```
es_api_basic_auth_username is not defined
```
?

Also I think having the conditions on separate lines is more readable if it is still possible to have that.